std_msgs/Float32 x
std_msgs/Float32 y
std_msgs/Float32 radius
std_msgs/UInt32 type
std_msgs/UInt32 number
